v1/services/storage/store.go

```diff
@@ -257,7 +257,11 @@ func (s *Store) tagKeysWithFieldPredicate(ctx context.Context, mqAttrs *metaquer
 	rs := reads.NewFilteredResultSet(ctx, mqAttrs.start, mqAttrs.end, cur)
 	for rs.Next() {
 		func() {
-			c := rs.Cursor()
+			c, err := rs.Cursor()
```

Comment:
Return the error out of this anonymous function; the top level function has an error return, so the plumbing is only local.  

The `rs.Next` loop should halt on error (as opposed to continuing as is should for `c == nil` when there is no data.